rxnorm-integration/pom.xml

```diff
@@ -64,12 +64,6 @@
             <version>1.0.0-SNAPSHOT</version>
             <scope>test</scope>
         </dependency>
-        <dependency>
```

Comment:
I believe the integration module requires the RxNormUtility class so I think this should stay.

Reply:
> I believe the integration module requires the RxNormUtility class so I think this should stay.

The dependency was declared twice. I removed the duplicate.